service/src/main.rs

```diff
@@ -677,38 +673,32 @@ fn execute_trusted_calls<E: Sidechain>(enclave_api: &E) {
 }
 
 /// gets a list of blocks that need to be synced, ordered from oldest to most recent header
-/// blocks that need to be synced are all blocks from the current header to the last synced header, iterating over parent
-fn get_blocks_to_sync(
+/// starting from the last synced head until the current head but at most BLOCK_SYNC_BATCH_SIZE.
```

Comment:
So the ordering is still preserved? We start at the oldest block and move towards the most recent one? I'm asking because my block import dispatching in the enclave currently relies on that.

Reply:
yes, the for loop counts upwards from the latest synced block to current head.